calculator.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Calculator</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width-device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie-edge">
+        <link rel="stylesheet" href="style.css">
+        <h1 id="Name" class="content">Number Calculator</h1>
+        <style type text"text/css">
+            label{
+    width:100px;
+    display:inline-block;
+    }
+        </style>
+    </head>
+    <body>
+        <script>
+        function AddNum(){
+            var val1 = parseInt(document.getElementById("value1").value);
+            var val2 = parseInt(document.getElementById("value2").value);
+            var ansD = document.getElementById("answer1");
+            ansD.value = val1+val2;
+        }    
+        function SubNum(){
+            var val3 = parseInt(document.getElementById("value3").value);
+            var val4 = parseInt(document.getElementById("value4").value);
+            var ansD = document.getElementById("answer2");
+            ansD.value = val3-val4;
+        }
+        function MultNum(){
+            var val5 = parseInt(document.getElementById("value5").value);
+            var val6 = parseInt(document.getElementById("value6").value);
+            var ansD = document.getElementById("answer3");
+            ansD.value = val5*val6;
+        }
+        function DivNum(){
+            var val7 = parseInt(document.getElementById("value7").value);
+            var val8 = parseInt(document.getElementById("value8").value);
+            var ansD = document.getElementById("answer4");
+            ansD.value = val7/val8;
+        }
+    </script>
```

Comment:
We've already discussed this but please create a separate JS file moving forward.